plugins/com.google.cloud.tools.eclipse.util/templates/appengine/web.xml.ftl

```diff
@@ -1,16 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
 <#if version == "2.5">
```

Comment:
It doesn't feel like you need an if here. Can you pass the values you want used directly?